patches/minecraft/net/minecraft/enchantment/Enchantment.java.patch

```diff
@@ -48,3 +59,20 @@
      public static void func_185257_f()
      {
          EntityEquipmentSlot[] aentityequipmentslot = new EntityEquipmentSlot[] {EntityEquipmentSlot.HEAD, EntityEquipmentSlot.CHEST, EntityEquipmentSlot.LEGS, EntityEquipmentSlot.FEET};
+@@ -209,4 +233,16 @@
+             return this.field_185275_e;
+         }
+     }
++
++    /* ======================================== FORGE START =====================================*/
```

Comment:
You're missing a space at the end of this. Should be like:

    /* ============ FORGE START ============ */

not

    /* ============ FORGE START ============*/